framework/integration/client/configmap/client.go

```diff
@@ -1,66 +1,56 @@
 // +build k8srequired
 
-package client
+package configmap
```

Comment:
Unsure why this shows up as a rename instead of a deletion and new file. Reading the `/nodeconfig/client.go` file first should hopefully make this file easier to read.